app/services/proofing/mock/ddp_mock_client.rb

```diff
@@ -32,12 +32,33 @@ class DdpMockClient < Proofing::Base
 
       proof do |applicant, result|
         result.transaction_id = TRANSACTION_ID
+
         response_body = File.read(
           Rails.root.join(
             'spec', 'fixtures', 'proofing', 'lexis_nexis', 'ddp', 'successful_response.json'
           ),
         )
-        result.review_status = case SsnFormatter.format(applicant[:ssn])
+        result.review_status = review_status(
+          session_id: applicant[:threatmetrix_session_id],
+          ssn: applicant[:ssn],
+        )
+
+        result.response_body = JSON.parse(
+          response_body.gsub('REVIEW_STATUS', result.review_status.to_s),
+        )
+      end
+
+      def review_status(session_id:, ssn:)
+        device_status = DeviceProfilingBackend.new.profiling_result(session_id)
+
+        case device_status
+        when 'no_result'
+          return nil
+        when 'reject', 'review', 'pass'
+          return device_status
+        end
+
+        case SsnFormatter.format(ssn)
```

Comment:
I vote for removing these magic SSNs and having the dropdown by the One True Way to trigger a certain device profiling result.

Reply:
cool I'll do that! My initial thought was to preserve any testing instructions we'd already sent people

Reply:
done in cbb1e17d0522744fec6a9c3b2737c75577014390